.testing/Makefile

```diff
@@ -128,15 +130,17 @@ REPORT_ERROR_LOGS ?=
 # Time measurement (configurable by the CI)
 TIME ?= time
 
-
 # Experiment configuration
-BUILDS ?= symmetric/MOM6 asymmetric/MOM6 openmp/MOM6
+EXECS ?= symmetric/MOM6 asymmetric/MOM6 openmp/MOM6
 CONFIGS ?= $(wildcard tc*)
-TESTS ?= grid layout rotate restart openmp nan $(foreach d,$(DIMS),dim.$(d))
 DIMS ?= t l h z q r
+TESTS ?= grid layout rotate restart openmp nan $(foreach d,$(DIMS),dim.$(d))
 
 # Default is to place work/ and results/ in current directory
-WORKSPACE ?= .
+BUILD ?= build
```

Comment:
For a user on gaea keeping the source in /home, I think this now requires providing two macros. Instead if you kept WORKSPACE and adding BUILD and WORK like this:
```
WORKSPACE ?= 
BUILD ?= $(WORKSPACE)build
WORK ?= $(WORKSPACE)work
```
You would retain the old functionality and provide the fine-grain control you want for the CI

Reply:
Is anyone else using `WORKSPACE`?  I am asking because I deliberately do not want to anchor these paths together in any way.

Reply:
I do, but I could also achieve the same with `config.mk` ...